GameData/Data.cs

```diff
@@ -9,6 +9,10 @@ namespace GameData
 {
     public class Data : IGameData
     {
+
+        public string PlayerName { get; set; } = "Eric";
```

Comment:
Can you make it so the GameData only supplies the default main player name and it's the engine that keeps track of what it is currently.